### PR TITLE
Update Get-CsOnlineUser.md - Minor changes 2

### PR DESCRIPTION
**Pull request by isrumnon**

### CODE REVIEW

**Review by get-itips — APPROVED** [review]

**Review by dariomws — COMMENTED** [review]
I think we should change the parameter type from "unlimited" to "UInt32" or "Int".

Then we should specify if the value could be anyone between 0 and 2147483647, or if it was increased/changed.

File: skype/skype-ps/skype/Get-CsOnlineUser.md [hunk 1]
```diff
@@ -329,11 +329,11 @@ In the Teams PowerShell Module version 3.0.0 or later, the format of the Assigne
 
 *Dropped Filter operators*:
 
-The following filter syntaxes are not supported in Teams PowerShell Module 3.0.0 and above:
+The following filter syntaxes have been modified in Teams PowerShell Module 3.0.0 and above:
```
Comment (JohanFreelancer9) [inline skype/skype-ps/skype/Get-CsOnlineUser.md:332]:
```suggestion
The following filter syntaxes have been modified in Teams PowerShell Module 3.0.0 and later:
```

File: skype/skype-ps/skype/Get-CsOnlineUser.md [hunk 2]
```diff
@@ -419,6 +419,9 @@ Accept wildcard characters: False
 ```
 
 ### -ResultSize
+
+*Note:* Starting with Teams PowerShell Modules version 4.0 or later "-ResultSize" type has  been changed to uint32
```
Comment (JohanFreelancer9) [inline skype/skype-ps/skype/Get-CsOnlineUser.md:423]:
```suggestion
**Note**: Starting with Teams PowerShell Modules version 4.0 and later "-ResultSize" type has  been changed to uint32.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @isrumnon, kindly also commit the minor suggested changes. Many thanks.

**Comment by yogkumgit** [comment]
@isrumnon

**Comment by isrumnon** [comment]
> @isrumnon

Done!

**Review by tseward — APPROVED** [review]

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@dariomws

**Review by dariomws — APPROVED** [review]

**Comment by yogkumgit** [comment]
@tiburd Please merge this PR. Thanks

**Comment by tiburd** [comment]
close > reopen

**Comment by isrumnon** [comment]
@tiburd please go ahead and merge. Thanks!

**Pull request merged**